src/image/README.md

```diff
@@ -4,6 +4,6 @@ Generating colorspace
 Source to the background image used primarly for the rgb colorspace.
 When making adjustments to this source, export the image as a png and run
 
-    openssl enc -base64 -in /path/to/rgb-colorspace.png | tr -d '\n' | pbcopy
+    openssl enc -base64 -in /path/to/rgb-colorpspace.png | tr -d '\n' | pbcopy
```

Comment:
typo!

Reply:
Haha, I changed this cause that's what the file names are called 🙊 I guess I should change the files to `rgb-colorspace.*` then?!

Reply:
🙈 hah! yeah